source/extensionPoints/util.py

```diff
@@ -95,7 +114,24 @@ def register(self, handler):
 		weak.handlerKey = key
 		self._handlers[key] = weak
 
-	def unregister(self, handler):
+	def moveToEnd(self, handler: HandlerT, last: bool = False) -> bool:

```

Comment:
considering by default this moves to the start, not the end, this name might be more general

```suggestion
	def moveToBoundary(self, handler: HandlerT, start: bool = True) -> bool:
```

Reply:
ah I see that `move_to_end` is a pythonic term. Feel free to ignore this suggestion